.circleci/config.yml

```diff
@@ -48,137 +38,25 @@ jobs:
-    steps:
-      - <<: *setup_kurtosis
-      - checkout
-
-      - run: kurtosis run ${PWD} "$(cat ./.circleci/tests/test-3.json)"
-
-  nightly_test_4:
-    executor: ubuntu_vm
-    resource_class: xlarge
-    steps:
-      - <<: *setup_kurtosis
-      - checkout
-
-      - run: kurtosis run ${PWD} "$(cat ./.circleci/tests/test-4.json)"
-
-  nightly_test_5:
-    executor: ubuntu_vm
-    resource_class: xlarge
-    steps:
-      - <<: *setup_kurtosis
-      - checkout
-
-      - run: kurtosis run ${PWD} "$(cat ./.circleci/tests/test-5.json)"
-
-  nightly_test_6:
-    executor: ubuntu_vm
-    resource_class: xlarge
-    steps:
-      - <<: *setup_kurtosis
-      - checkout
-
-      - run: kurtosis run ${PWD} "$(cat ./.circleci/tests/test-6.json)"
-
-  nightly_test_7:
-    executor: ubuntu_vm
-    resource_class: xlarge
-    steps:
-      - <<: *setup_kurtosis
-      - checkout
-
-      - run: kurtosis run ${PWD} "$(cat ./.circleci/tests/test-7.json)"
-
-  nightly_test_8:
-    executor: ubuntu_vm
-    resource_class: xlarge
-    steps:
-      - <<: *setup_kurtosis
-      - checkout
-
-      - run: kurtosis run ${PWD} "$(cat ./.circleci/tests/test-8.json)"
-
 workflows:
-  check_latest_version:
-    # TODO rever this to when: << pipeline.parameters.should-enable-check-latest-version-workflow > before PR merge
-    when: << pipeline.parameters.should-enable-build-workflow >>
-    jobs:
-      - check_latest_version:
-          context:
-            - slack-secrets
-      - nightly_test_1:
-          context:
-            - slack-secrets
-      - nightly_test_2:
-          context:
-            - slack-secrets
-      - nightly_test_3:
-          context:
-            - slack-secrets
-      - nightly_test_4:
-          context:
-            - slack-secrets
-      - nightly_test_5:
-          context:
-            - slack-secrets
-      - nightly_test_6:
-          context:
-            - slack-secrets
-      - nightly_test_7:
-          context:
-            - slack-secrets
-      - nightly_test_8:
-          context:
-            - slack-secrets
-
-  build:
-    when: << pipeline.parameters.should-enable-build-workflow >>
+  version: 2
+  nightly_tests_workflow:
     jobs:
-      # -- PR check jobs ------------------------------------------
-      - kurtosis-docs-checker/check-docs:
-          should-check-changelog: false
-          markdown-link-check-config-json: |
-            {
-                "ignorePatterns": [
-                    {
-                        "pattern": "https://github.com/kurtosis-tech/eth2-package"
-                    }
-                ]
-            }
-          filters:
-            branches:
-              ignore:
-                - main
-      - run_starlark:
-          context:
-            # This pulls in KurtosisBot's Github token, so that we can read from our private repos
-            - github-user
-          filters:
-            branches:
-              ignore:
-                - main
+      - run_starlark
```

Comment:
adding this comment so i don't forget but this test should be nightly so behind the 

`when: << pipeline.parameters.should-enable-check-latest-version-workflow >>` flag 

don't worry about it now. this is anote to self